known-build-systems.json

```diff
@@ -1,13 +1,12 @@
 [
-    "poetry",
-    "poetry-core",
-    "flit",
-    "flit-core",
-    "pbr",
-    "flitBuildHook",
```

Comment:
It's kinda hard to tell that this is the only line which changed without ignoring whitespace. Would it be possible to either move the whitespace change to a separate commit or omit it entirely?